ideascube/serveradmin/backup.py

```diff
@@ -113,6 +128,11 @@ def delete(self):
 
     @classmethod
     def create(cls, format=None):
+        format = format or Backup.FORMAT
+        if format not in Backup.SUPPORTED_FORMATS_AT_CREATION:
+            raise ValueError(
+                _("Format {} is not supported at creation").format(format)
+            )
         name = make_name(format)
         backup = Backup(name)
```

Comment:
Yes, we should do some refactoring.

In another pr maybe ? To avoid a too big pr here.